Tensile/KernelWriterAssembly.py

```diff
@@ -1113,28 +1113,27 @@ def initKernel(self, kernel, tPA, tPB ):
 
     # registers per element
     self.bpr = 4 # all registers are 32bit
-    self.bpeAB = int(self.bpr*\
-        kernel["ProblemType"]["DataType"].numRegisters())
-    self.bpeCexternal = int(self.bpr*\
-        kernel["ProblemType"]["DataType"].numRegisters())
+    self.bpeAB = int(self.bpr * kernel["ProblemType"]["DataType"].numRegisters())
+    self.bpeCexternal = int(self.bpr * kernel["ProblemType"]["DataType"].numRegisters())
+    self.bpeDexternal = int(self.bpr * kernel["ProblemType"]["DataType"].numRegisters())
+
     #jgolds Need to check device for support
     if kernel["ProblemType"]["HighPrecisionAccumulate"]:
         if kernel["ProblemType"]["DataType"].isHalf():
```

Comment:
Can perhaps combine half and bfloat 16 cases as they are identical

Reply:
done